docs/install-cpu-source.md

```diff
@@ -1,11 +1,10 @@
 ```console
 pip install -U pip
 pip install -U setuptools wheel
-pip install -U uv
 
 # CPU version of pytorch has smaller footprint - see installation instructions in
 # pytorch documentation - https://pytorch.org/get-started/locally/
-uv pip install torch==2.4.1+cpu torchvision==0.19.1+cpu --index-url https://download.pytorch.org/whl/cpu
+pip install torch==2.4.1+cpu torchvision==0.19.1+cpu --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Why remove uv for cpu source install? We use uv anyways in the `full_install.sh` script. I'd recommend using UV for the torch install as well. Can also remove `pip install -U pip` and `pip install -U setuptools wheel`

```
pip install -U uv
python -m uv pip install torch==....
```

Reply:
Simplified